diff -ur php70.o/php-fpm.d/www.conf php70/php-fpm.d/www.conf
--- php70.o/php-fpm.d/www.conf       2015-10-14 19:08:56.645457712 +0000
+++ php70/php-fpm.d/www.conf 2015-10-14 19:25:03.065407480 +0000
@@ -20,7 +20,7 @@
 ; Note: The user is mandatory. If the group is not set, the default user's group
 ;       will be used.
 ; RPM: apache Choosed to be able to access some dir as httpd
-user = apache
+user = festink
 ; RPM: Keep a group allowed to write in log dir.
 group = apache

diff -ur php70.o/php.ini php70/php.ini
--- php70.o/php.ini  2015-10-14 19:08:56.641457712 +0000
+++ php70/php.ini    2015-10-14 19:19:01.126426292 +0000
@@ -363,7 +363,7 @@
 ; threat in any way, but it makes it possible to determine whether you use PHP
 ; on your server or not.
 ; http://php.net/expose-php
-expose_php = On
+expose_php = Off

 ;;;;;;;;;;;;;;;;;;;
 ; Resource Limits ;
